### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by avtikhon**
- Added new tags and rolling versions.
- Reorganized tables of versions to make it compact.
- Corrected patterns for stable versions set.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -373,18 +356,36 @@ All images are pushed to [Docker Hub](docker_hub_tags).
...
 - `x.y` == `x.y.<last-z>` (`==` means 'points to the same image')
 - `1` == `1.<last-y>.<last-z>`
 - `2` == `2.<last-y>.<last-z>`
 - `latest` == `2`
 
+Examples of stable rolling versions:
+
+- `1` -> `1.<last-y>.2`
+- `2` -> `2.<last-y>.2`
+- `latest` -> `2.<last-y>.2`
```
Comment (Totktonada) [inline README.md:377]: It seems it feeds the save information as the paragraps above.

File: README.md [hunk 2]
```diff
@@ -373,18 +356,36 @@ All images are pushed to [Docker Hub](docker_hub_tags).
...
 Special builds (CentOS) are updated with the same policy as rolling versions:
 
 - `1.x-centos7` image offers a last `1.<last-y>.<last-z>` release
 - `2.x-centos7` image offers a last `2.<last-y>.<last-z>` release
 
+Examples of stable rolling versions for CentOS 7 images:
+
+- `1.x-centos7` -> `1.<last-y>.2`
+- `2.x-centos7` -> `2.<last-y>.2`
+
```
Comment (Totktonada) [inline README.md:388]: Same here. Looks the same as the paragraph above.

File: README.md [hunk 3]
```diff
@@ -373,18 +356,36 @@ All images are pushed to [Docker Hub](docker_hub_tags).
 
 Fixed version tags (`x.y.z`) are frozen: we never update them.
 
+Example of minor versions timeline:
+
+- `x.y.0` - Alpha
+-   | -> `x.y.1` - Beta
+-     | -> `x.y.2` - Stable
+-       | -> `x.y.3` - Update
```
Comment (Totktonada) [inline README.md:364]: It is poorly rendered into HTML. I guess this way it will work smoothly:

```
- a
  - b
    - c
      - d
```

Comment (Totktonada) [inline README.md:364]: We have no 'update' maturity status in our [docs](https://www.tarantool.io/en/doc/latest/dev_guide/release_management/). I would mark it just as Stable, the same as .2.

File: README.md (hunk 1 above)
Comment (Totktonada) [reply README.md:377]: My bad. After discussion with Alexander I see the difference. Here we have explicit `.2` at the end.

I vote to just replace the `.<last-z>` with `.2` sbove.

File: README.md (hunk 2 above)
Comment (Totktonada) [reply README.md:388]: Same as above: I would just replace `.<last-z>` with `.2`. My mistake.

File: README.md [hunk 4]
```diff
@@ -373,17 +356,24 @@ All images are pushed to [Docker Hub](docker_hub_tags).
...
 Fixed version tags (`x.y.z`) are frozen: we never update them.
 
+Example of minor versions timeline:
+
+- `x.y.0` - Alpha
+  - `x.y.1` - Beta
+    - `x.y.2` - Stable
+      - `x.y.3` - Stable
+
 Rolling versions are updated to the last fixed version tags:
```
Comment (Totktonada) [inline README.md:366]: Last → last stable.

File: README.md [hunk 5]
```diff
@@ -321,43 +321,26 @@ Fixed versions:
...
-| 1.10.2     | dockerfile/alpine_3.5 |
-| 1.10.3     | dockerfile/alpine_3.5 |
-| 1.10.4     | dockerfile/alpine_3.9 |
-| 1.10.5     | dockerfile/alpine_3.9 |
-| 1.10.6     | dockerfile/alpine_3.9 |
-| 1.10.7     | dockerfile/alpine_3.9 |
-| 2.1.0      | dockerfile/alpine_3.5 |
-| 2.1.1      | dockerfile/alpine_3.5 |
-| 2.1.2      | dockerfile/alpine_3.5 |
+| 1.10.[0-3] | dockerfile/alpine_3.5 |
```
Comment (Totktonada) [inline README.md:324]: To be honest, I'm not sure about this change. Regex-free text is easier to read.

Can we ask someone else, how it looks better?

Comment (Totktonada) [reply README.md:324]: Or we can split the long table to several shorter ones.

**Review by Totktonada — APPROVED** [review]
Most of my complains were resolved and so I'm mostly okay.

**Comment by Totktonada** [comment]
Timur, please, glance if time permits.

File: README.md (hunk 5 above)
Comment (tsafin) [reply README.md:324]: I beg to disagree - those regexps make this table less verbose and easier to parse.

Comment (tsafin) [reply README.md:324]: Though we could put the same infomarmation as ranges, i.e.
`1.10.0 .. 1.10.3`

File: README.md [hunk 6]
```diff
@@ -373,17 +356,24 @@ All images are pushed to [Docker Hub](docker_hub_tags).
...
-- `2` == `2.<last-y>.<last-z>`
+- `1` == `1.<last-y>.2`
+- `2` == `2.<last-y>.2`
 - `latest` == `2`
 
 Special builds (CentOS) are updated with the same policy as rolling versions:
 
-- `1.x-centos7` image offers a last `1.<last-y>.<last-z>` release
-- `2.x-centos7` image offers a last `2.<last-y>.<last-z>` release
+- `1.x-centos7` image offers a last `1.<last-y>.2` release
```
Comment (tsafin) [inline README.md:375]: This is confusing as a hell. But after 2nd thought I understood it!

**Review by tsafin — APPROVED** [review]
In general I'm ok

File: README.md (hunk 5 above)
Comment (Totktonada) [reply README.md:324]: Okay so. Ranges looks better for me. Anyway, Alexander, please pick up any choice you consider as the best.

File: README.md (hunk 6 above)
Comment (Totktonada) [reply README.md:375]: Any proposal? Or it'll out of scope here?

Comment (tsafin) [reply README.md:375]: Nope. No proposals :)

P.S.
When you see magical `.2` it confuses a lot...

**Pull request merged**